samples/update_column.bal

```diff
@@ -0,0 +1,27 @@
+import ballerina/log;
```

Comment:
Licence header should be added in the top of all the .bal files

Reply:
Do we need to give in samples files also?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/commit/17d011bedace29c65f66e3fc2257cb63668c17fb